packages/state-manager/src/sagas/communities/communities.slice.ts

```diff
@@ -22,6 +21,33 @@ export class CommunitiesState {
   public psk: string | undefined
 }
 
+// TODO: remove after setting strict in 'desktop' and 'mobile' packages
+export interface Community {
```

Comment:
I suppose this is a leftover after resolving some conflicts? I moved Community interface to 'types' package a while ago

Reply:
Yea it didn't look like this interface is being used. I can check again and remove it probably.

Reply:
Oh, I see that this PR adds this back, how funny. Not sure how that got in there. Maybe when I rebased.